src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,10 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+18.0.0b3
```

Comment:
Please choose a new major version like `19.0.0b1` as you're making a breaking change.

https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md